source/_posts/2021-05-05-release-20215.markdown

```diff
@@ -503,7 +506,7 @@ The following attributes are no longer available:
 
 {% details "Shelly" %}
 
-The Shelly Wi-fi RSSI sensor unit was wrong, it is **dBm** and not **dB**.
+The Shelly Wifi RSSI sensor unit was wrong, it is **dBm** and not **dB**.
```

Comment:
This is actually incorrect.

```suggestion
The Shelly Wi-Fi RSSI sensor unit was wrong, it is **dBm** and not **dB**.
```

See:
- <https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi>
- <https://developers.google.com/style/word-list#letter-w>